<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1659</width>
    <height>1108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWebEngineView" name="webEngineView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>1301</width>
     <height>1011</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>http://localhost:8888/</string>
    </url>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>1030</y>
     <width>891</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>1070</y>
     <width>458</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_forward">
      <property name="text">
       <string>Forward</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_reload">
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_stop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveAnno">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>900</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openObjectDirButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="gistLogo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ailabLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openDataDirButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>640</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QTreeView" name="dataDirSelecter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>670</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logger">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>940</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="objectDirSelecter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string> Data Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Object Lists</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoggerTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>900</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Logger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectAnnoDataButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>640</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAnnoObjectButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTreeView" name="objectDirSelecter_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Annotation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAnnoObjectButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAnnoObjectButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refine</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
